continuous_integration/hdfs/README.md

```diff
@@ -0,0 +1,59 @@
+## HDFS testing on Travis CI
+
+Dask & HDFS testing relies on a docker container. The tests are setup to run on
+Travis CI, but only under the following conditions:
+
+- Merges to master
+- PRs where the branch name contains the string `"__TEST_HDFS__"`
+
+If you make a PR changing HDFS functionality it'd be good to have the HDFS
+tests run, please add `"__TEST_HDFS__"` to the end of your branch name.
+
+If you've already made a PR (and thus can't change the branch name), you can
+temporarily comment out the `if:` configuration in the `.travis.yml` for the
+HDFS tests.
+
+## Setting up HDFS testing locally using Docker
+
+Assumes docker is already installed and the docker-daemon is running.
+
+From the root directory in the repo:
+
+- First get the docker container:
+
+```bash
+# Either pull it from docker hub
+docker pull daskdev/dask-hdfs-testing
```

Comment:
This is the same image as above? Maybe point out the dockerfile in the repo here.
Should we mention that it is enough to have any hdfs running (and there are lots of ways) to run tests as normal, but the dockerfile makes the difference of running them *within* the same container as docker (i.e., on a specific linux).

Reply:
The tests are written in a way that assumes this setup. Other setups probably won't work for our tests.